packages/core/phase2/lib/getOperationSequence/areAllResultsAlreadyPresentOnPnc/isMatchToPncAdjAndDis/createPncAdjudicationByAho.ts

```diff
@@ -5,7 +5,7 @@ import type { PncAdjudication } from "../../../../../types/PncQueryResult"
 import isRecordableResult from "../../../isRecordableResult"
 import createPncAdjudication from "./createPncAdjudication"
 
-const getAdjFromAho = (results: NonEmptyArray<Result>, hearingDate: Date): PncAdjudication => {
+const createPncAdjudicationByAho = (results: NonEmptyArray<Result>, hearingDate: Date): PncAdjudication => {
```

Comment:
I like that we've renamed this better and used the full word of adjudication, but I think it should be `createPncAdjudicationFromAho` instead? 🤔